packages/card/src/uxp-card.css

```diff
@@ -11,3 +11,145 @@ governing permissions and limitations under the License.
 */
 
 /* write uxp style overrides */
+
+:host {
```

Comment:
@inasirhussain I want to understand each of the style overrides in this file. Do you have tracking tickets or wiki?

Reply:
@deepthi-majji - This all overrides are to polyfill the various missing logical properties. I can do one thing, put the UXP JIRA ticket tracking the logical property support.